cats/__init__.py

```diff
@@ -7,15 +7,20 @@
 import sys
 
 from .timeseries_conversion import cat_converter  # noqa: F401
-from .api_query import get_tuple  # noqa: F401
+from .api_query import get_tuple
```

Comment:
At some point I expect we'll want to rename get_tuple to something else. But okay for now

Reply:
Agreed -  I've been trying to resist the urge of making too many changes at once.